### PR TITLE
docs: restructuring docs and rewording some stuff

### PR DESCRIPTION
**Pull request by VincentRPS**

### CODE REVIEW

File: examples/basic-ipc/webserver.py [hunk 1]
```diff
@@ -8,11 +8,11 @@
...
 @app.route("/")
 async def index():
-    member_count = await ipc_client.request(
-        "get_member_count", guild_id=12345678
-    )  # get the member count of server with ID 12345678
-
-    return str(member_count)  # display member count
+    # get the member count of the guild with the id 12345678
+    member_count = await ipc_client.request("get_member_count", guild_id=12345678)
+
```
Comment (DenverCoder1) [inline examples/basic-ipc/webserver.py:13]: Remove this trailing whitespace (probably will fix the linter issue)

```suggestion

```

**Review by DenverCoder1 — APPROVED** [review]
Looks good. :+1:

The formatting of the examples I'll probably clean up more later anyway.

**Comment by DenverCoder1** [comment]
Docs are showing some warnings

```diff
$ make -C docs html
Running Sphinx v4.2.0
making output directory... done
loading intersphinx inventory from https://docs.aiohttp.org/en/stable/objects.inv...
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://nextcord.readthedocs.io/en/latest/objects.inv...
- WARNING: failed to reach any of the inventories with the following issues:
- intersphinx inventory 'https://docs.aiohttp.org/en/stable/objects.inv' not fetchable due to <class 'requests.exceptions.ConnectionError'>: HTTPSConnectionPool(host='docs.aiohttp.org', port=443): Max retries exceeded with url: /en/stable/objects.inv (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff2d0e57a90>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 5 source files that are out of date
updating environment: [new config] 5 added, 0 changed, 0 removed
reading sources... [100%] index                                                                                                                                        
- WARNING: autodoc: failed to import class 'ipc.client.Client' from module 'nextcord.ext'; the following exception was raised:
- No module named 'nextcord.ext.ipc'
- WARNING: autodoc: failed to import class 'ipc.errors.IPCError' from module 'nextcord.ext'; the following exception was raised:
- No module named 'nextcord.ext.ipc'
- WARNING: autodoc: failed to import class 'ipc.errors.NoEndpointFoundError' from module 'nextcord.ext'; the following exception was raised:
- No module named 'nextcord.ext.ipc'
- WARNING: autodoc: failed to import class 'ipc.errors.ServerConnectionRefusedError' from module 'nextcord.ext'; the following exception was raised:
- No module named 'nextcord.ext.ipc'
- WARNING: autodoc: failed to import class 'ipc.errors.JSONEncodeError' from module 'nextcord.ext'; the following exception was raised:
- No module named 'nextcord.ext.ipc'
- WARNING: autodoc: failed to import class 'ipc.errors.NotConnected' from module 'nextcord.ext'; the following exception was raised:
- No module named 'nextcord.ext.ipc'
- WARNING: autodoc: failed to import function 'ipc.server.route' from module 'nextcord.ext'; the following exception was raised:
- No module named 'nextcord.ext.ipc'
- WARNING: autodoc: failed to import class 'ipc.server.Server' from module 'nextcord.ext'; the following exception was raised:
- No module named 'nextcord.ext.ipc'
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] index                                                                                                                                         
generating indices... genindex done
writing additional pages... search done
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 9 warnings.
```

**Comment by VincentRPS** [comment]
> Docs are showing some warnings
> 
> ```diff
> $ make -C docs html
> Running Sphinx v4.2.0
> making output directory... done
> loading intersphinx inventory from https://docs.aiohttp.org/en/stable/objects.inv...
> building [mo]: targets for 0 po files that are out of date
> building [html]: targets for 5 source files that are out of date
> updating environment: [new config] 5 added, 0 changed, 0 removed
> reading sources... [100%] index                                                                                                                                        
> - WARNING: autodoc: failed to import class 'ipc.client.Client' from module 'nextcord.ext'; the following exception was raised:
> - No module named 'nextcord.ext.ipc'
> - WARNING: autodoc: failed to import class 'ipc.errors.IPCError' from module 'nextcord.ext'; the following exception was raised:
> - No module named 'nextcord.ext.ipc'
> - WARNING: autodoc: failed to import class 'ipc.errors.NoEndpointFoundError' from module 'nextcord.ext'; the following exception was raised:
> - No module named 'nextcord.ext.ipc'
> - WARNING: autodoc: failed to import class 'ipc.errors.ServerConnectionRefusedError' from module 'nextcord.ext'; the following exception was raised:
> - No module named 'nextcord.ext.ipc'
> - WARNING: autodoc: failed to import class 'ipc.errors.JSONEncodeError' from module 'nextcord.ext'; the following exception was raised:
> - No module named 'nextcord.ext.ipc'
> - WARNING: autodoc: failed to import class 'ipc.errors.NotConnected' from module 'nextcord.ext'; the following exception was raised:
> - No module named 'nextcord.ext.ipc'
> - WARNING: autodoc: failed to import function 'ipc.server.route' from module 'nextcord.ext'; the following exception was raised:
> - No module named 'nextcord.ext.ipc'
> - WARNING: autodoc: failed to import class 'ipc.server.Server' from module 'nextcord.ext'; the following exception was raised:
> - No module named 'nextcord.ext.ipc'
> looking for now-outdated files... none found
> pickling environment... done
> checking consistency... done
> preparing documents... done
> writing output... [100%] index                                                                                                                                         
> generating indices... genindex done
> writing additional pages... search done
> copying static files... done
> copying extra files... done
> dumping search index in English (code: en)... done
> dumping object inventory... done
> build succeeded, 8 warnings.
> ```

yeah idk i will try to fix it ig

**Comment by VincentRPS** [comment]
@DenverCoder1 i can't find any errors i honestly don't know what is causing that error maybe ask ooliver

**Comment by DenverCoder1** [comment]
@VincentRPS Add this at the top of conf.py

```
import sys
import os

sys.path.insert(0, os.path.abspath('..'))
```

**Comment by VincentRPS** [comment]
> @VincentRPS Add this at the top of conf.py. Fixed it for me.
> 
> ```
> import sys
> import os
> 
> sys.path.insert(0, os.path.abspath('..'))
> ```

Okay i added it to conf now

File: docs/ext/ipc/client.rst [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
-.. currentmodule:: nextcord.ext.ipc.client
-
-.. autoclass:: Client
-    :members:
+Client Connection
+=================
+
+The IPC client is very simple.
+It will simply connect to your server process and send JSON data.
+If you do not supply a port on initialisation, the client will connect to the multicast :doc:`server` and return the port from said server.
```
Comment (DenverCoder1) [inline docs/ext/ipc/client.rst:6]: Link text shows up as "Server Setup", should be "server"

image

```suggestion
If you do not supply a port on initialisation, the client will connect to the multicast :doc:`server<server>` and return the port from said server.

```

File: docs/conf.py [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
+import os
+
 
 project = "nextcord-ext-ipc"
-copyright = "2021, Nextcord Developers"
-author = "Nextcord Developers"
+copyright = "2021, nextcord"
+author = "nextcord"
+
+sys.path.insert(0, os.path.abspath('..'))
```
Comment (DenverCoder1) [inline docs/conf.py:10]: Black wants double quotes here
```suggestion
sys.path.insert(0, os.path.abspath(".."))
```

**Review by DenverCoder1 — APPROVED** [review]

**Pull request merged**